hooks/BlueprintCheckAdditionalField.cpp

```diff
@@ -0,0 +1,15 @@
+#define SECTION(index, address) ".section h"#index"; .set h"#index","#address";"
+#include "../define.h"
+asm(
+    
+    SECTION(1, 0x004CF7F2)
+    "jmp "QU(GetBpFieldValues)";"
+    
+    SECTION(2, 0x004CF7F7)
```

Comment:
If the two sections are that close, for the amount of reading each new section takes up, I'd rather only have one.